aksel.nav.no/website/sanity/schema/objects/shared/riktekst.tsx

```diff
@@ -65,10 +65,19 @@ export const block = {
         component: ({ children }) => <InlineCode>{children}</InlineCode>,
       },
       {
-        title: "KBD",
+        title: "Quote",
+        value: "quote",
+        icon: () => (
+          <span className="font-semibold" aria-label="quote">
+            Q
+          </span>
+        ),
+        component: ({ children }) => <q>{children}</q>,
+      },
+      {
+        title: "Keyboard",
         value: "kbd",
-        icon: () => <KBD>KBD</KBD>,
-
+        icon: () => <kbd>KBD</kbd>,
```

Comment:
Was this change intended?
![image](https://github.com/navikt/aksel/assets/83693529/b6741c37-c545-4a49-8b3e-332a4bb6aa43) -> ![image](https://github.com/navikt/aksel/assets/83693529/ce2ff318-9866-4f4e-9f6f-f0f98215d332)

Reply:
Yes, think its a little more in line with how the rest of the richtext editor shows `marks`